docs/buildah-run.1.md

```diff
@@ -58,6 +58,12 @@ not disabled.
 List of directories in which the CNI plugins which will be used for configuring
 network namespaces can be found.
 
+**--contextdir** *directory*
+
+Allows setting context directory for current RUN invocation. Specifying a context
+directory causes Buildah to chroot into that context directory. This means copying
+files pointed at by symbolic links outside of the chroot will fail.
```

Comment:
This still needs correcting - we don't chroot here, and the specified location is used as the root directory for source locations specified in `--mount` options that specify a mount type of "bind".